src/SignalRServiceExtension/Bindings/LiteServiceEndpoint.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.SignalR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class LiteServiceEndpoint
```

Comment:
>LiteServiceEndpoint [](start = 17, length = 19)

Why call it Lite?

Reply:
It only contains a subset of members of `ServiceEndpoint` needed for routing.

Reply:
how about call it `ServiceEndpoint`? looks like server sdk's `ServiceEndpoint` doesn't expose to cx, so we can use namespace to tell them internally, cx use `ServiceEndpoint` is ok

---
In reply to: [571564306](https://github.com/Azure/azure-functions-signalrservice-extension/pull/161#discussion_r571564306) [](ancestors = 571564306)

Reply:
`ServiceEndpoint` is public.